File: instrasoft-api/src/main/java/co/com/foundation/intersoft/domain/Area.java
package co.com.foundation.intersoft.domain;

import java.util.List;

import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.Setter;

@Document(collection = "area")
@Getter
@Setter
@Builder
@AllArgsConstructor
public class Area {

	@Id
	private long _id;
	private String name;
	private City city;
	private List<Employee> employees;
	private List<Active> actives;
}

File: intrasoft-ui/src/app/domain/area.ts
import { City } from "./city";
import { Employee } from "./employee";
import { Active } from "./active";
import { Inventory } from "./inventory";

export class Area{

    _id: String;
    name: String;
    city: City;
    employees: Array<Employee>;
    actives: Array<Active>;
    inventories: Array<Inventory>;

}